common/config/rush/common-versions.json

```diff
@@ -71,10 +71,6 @@
       "^4.2.0",
       "^4.0.2",
     ],
-    "@openzeppelin/contracts": [
-      "~3.4.2",
-      "~4.3.2"
```

Comment:
3.4.2 for `solidity ^0.6.0`
4.4.2 for `solidity ^0.8.0`
both are required here